client/src/components/CreateAdForm.js

```diff
@@ -31,6 +30,7 @@ export const CreateAdForm = (props) => {
   const submit = (e) => {
     e.preventDefault();
     handleSubmit(ad);
+    e.target.reset();
```

Comment:
the form in the browser may reset with this, but does the actual state of "ad" reset?

Reply:
what happens if submit fails? the user's info would be cleared and would have to type it all over again before resubmitting

Reply:
Good point. I make the reset happen only after successful submission.